targets/targets.json

```diff
@@ -2657,6 +2657,19 @@
             "STM32G431xx"
         ]
     },
+    "NUCLEO_G431RB": {
+        "inherits": [
+            "MCU_STM32G431xB"
+        ],
+        "detect_code": [
+            "0850"
+        ],
+        "overrides": {
+            "clock_source" : "USE_PLL_HSE_EXTC|USE_PLL_HSE_XTAL",
```

Comment:
Maybe remove this line (and keep with the default value like for NUCLEO_G474RE)

Reply:
No dice. Removing that and inheriting the default `clock_source=USE_PLL_HSE_EXTC` from the `MCU_STM32G4` target, has the same problem. So does `USE_PLL_HSI`.

Reply:
```suggestion
```